package.json

```diff
@@ -18,6 +18,7 @@
     "lint": "eslint --fix app/assets/javascripts",
     "tsc": "tsc --project app/assets/javascripts/tsconfig.json",
     "test": "jest --config app/assets/javascripts/jest.config.js",
+    "test:coverage": "yarn test --coverage",
```

Comment:
This is failing for me. Is it ready to be used?

Reply:
It works on my side. Once PR is cleaned up, we can check it again.